src/js/components/HeaderBar.tsx

```diff
@@ -1,20 +1,67 @@
 import * as React from "react";
+import AuthStore from "#SRC/js/stores/AuthStore";
+
 import { HeaderBar as UIHeaderBar } from "ui-kit-stage/HeaderBar";
+import AccountDropdown from "./AccountDropdown";
+
+function getUserAccountDropdown() {
+  const menuItems = [
+    {
+      html: "Logout",
+      id: "logout",
+      onClick: AuthStore.logout
+    }
+  ];
+
+  var userLabel = AuthStore.getUserLabel();
+  return (
+    <AccountDropdown
+      menuItems={menuItems}
+      userName={userLabel}
+      willAnchorRight={false}
+    />
+  );
+}
+
+function getClusterDropdown() {
+  const menuItems = [
+    {
+      html: "Install CLI",
+      id: "install-cli",
+      onClick: AuthStore.logout
+    }
+  ];
+
+  const userLabel = "frontend-us-west-2-bit1j110bd";
+
+  return (
+    <AccountDropdown
+      menuItems={menuItems}
+      userName={userLabel}
+      willAnchorRight={true}
+    />
+  );
+}
 
 export default function HeaderBar() {
   // remove this to activate component
   if (arguments) {
     return null;
   }
 
+  var userAccountDropdown = getUserAccountDropdown();
```

Comment:
Can we use `const` instead of `var` since we are not mutating those two values?

Reply:
👍